package.json

```diff
@@ -147,5 +139,5 @@
       "path": "packages/pharos/lib/index.js"
     }
   ],
-  "packageManager": "yarn@1.22.19"
+  "packageManager": "yarn@4.0.2"
```

Comment:
This isn't right

Reply:
Let's see if we can revert all the Yarn-related items for the moment.